consensus/istanbul/backend/announce.go

```diff
@@ -111,7 +209,7 @@ func (sb *Backend) announceThread() {
 			logger.Debug("Announce version is not newer than the existing version", "existing version", sb.announceVersion, "attempted new version", version)
 			return
 		}
-		if err := sb.setAndShareUpdatedAnnounceVersion(version); err != nil {
+		if err := sb.announceManager.setAndShareUpdatedAnnounceVersion(version); err != nil {
```

Comment:
i would expect this whole function to be part of the announce manager.. maybe on a later PR

Reply:
Yes, actually it has already been moved in PR # 2